test/transition.js

```diff
@@ -3,20 +3,15 @@ var transition = require('../src/transition')
 var props = transition.properties;
 
 describe('Transition helpers', () => {
-
-  //beforeEach(()=>{
-  //   document.body.innerHTML = window.__html__['test/fixtures/event.html']
-  // })
-
   it('should parse duration from node property', () => {
     var el = document.createElement('div');
 
-    el.style[props.duration] = '1.4s'
+    el.style[props.duration] = '2s'
 
-    expect(transition.end._parseDuration(el)).to.equal(1400)
+    expect(transition.end._parseDuration(el)).to.equal(2000)
```

Comment:
> The original test should work, If it's failing its an indication that something broke

Yes, indeed. Do you want me to [skip it](https://github.com/react-bootstrap/dom-helpers/blob/master/test/query.js#L133) ?